packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -187,3 +201,16 @@ class ${class_name}:
 
         for response in self._get_stub().Measure(request):
             yield self._deserialize_response(response)
+
+    def register_pin_map(self, pin_map_path: str) -> str:
+        """Registers the pin map with the pin map service.
+        
+        Args:
+            pin_map_path: Absolute path of the pin map file.
+
+        Returns:
+            Registered pin map id.
+        """
+        self.pin_map_id = self.pin_map_client.update_pin_map(pin_map_path)
+
+        return self.pin_map_id
```

Comment:
Do we have any need to return the pin_map_id?

Reply:
No, not necessary. Hence removed.